xarray/core/formatting_html.py

```diff
@@ -213,6 +247,13 @@ def array_section(obj):
     expand_option_name="display_expand_data_vars",
 )
 
+index_section = partial(
+    _mapping_section,
+    name="Indexes",
+    details_func=summarize_indexes,
+    max_items_collapse=15,
```

Comment:
Do we want to collapse the indexes section by default, regardless of the number of items?

The list of names is redundant with the coordinates section, and it is probably fine if this section is one-click away (it's not primary information).